test/gtest/ucs/test_rcache.cc

```diff
@@ -68,7 +69,7 @@ class test_rcache : public ucs::test {
             sizeof(region),
             UCS_PGT_ADDR_ALIGN,
             ucs_get_page_size(),
-            UCM_EVENT_VM_UNMAPPED,
+            UCM_EVENT_VM_UNMAPPED | UCM_EVENT_MEM_TYPE_FREE,
```

Comment:
Should we use `UCM_LEVEL_MEM_TYPE_FREE` in rcache tests? On one hand we should, because actual use cases of rcache (e.g., IB, gdrcopy) register that event. On the other hand, not using it allows us to test rcache under circumstances similar to when the CUDA runtime is linked statically.
Or maybe we should repeat the tests with and without `UCM_EVENT_MEM_TYPE_FREE`.